CRM/Admin/Form/Preferences/Display.php

```diff
@@ -37,6 +37,7 @@
 class CRM_Admin_Form_Preferences_Display extends CRM_Admin_Form_Preferences {
   public function preProcess() {
     CRM_Utils_System::setTitle(ts('Settings - Display Preferences'));
+    $optionValues = CRM_Activity_BAO_Activity::buildOptions('activity_type_id', 'validate');
```

Comment:
I don't think you want to pass the 'validate' context here because then labels won't be shown properly.

Reply:
Updated the changes as per the above comment. Also handled the false display of notification description text(both activity and case activity forms) for activity types selected in  `do not notify` setting. Thanks for the review @colemanw 

![image](https://user-images.githubusercontent.com/5929648/33700548-e8f88328-db3f-11e7-83a6-f53db99f7881.png)

Reply:
Ah, missed to rebase before pushing the update which removed your commit. I've now included the js cleanup you did on this PR.